configure.ac

```
@@ -844,7 +844,9 @@ if test x"$with_libjxl" != x"no"; then
      AS_IF([test x"$with_libjxl_module" = x"no"],
        [PACKAGES_USED="$PACKAGES_USED libjxl"])
     ],
-    [AC_MSG_WARN([libjxl not found; disabling libjxl support])
+    [AS_IF([test x"$with_libjxl" = x"test"],
+       AC_MSG_WARN([libjxl not found; disabling libjxl support]),
+       AC_MSG_ERROR([libjxl not found]))
```

Perhaps we should mention what the reason for the error is?
```suggestion
       AC_MSG_ERROR([libjxl explicitly requested, but a suitable version was not found]))
```